src/Pages/PricePage/PricePage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+interface Props{}
+
+
+const PricePage = (props:Props)=>{
+    return <div className='nav'>Pricing </div>
```

Comment:
you dont really have .nav class defined anywhere, so it does exactly nothing

Reply:
Deleted